include/net/ieee802154_radio.h

```diff
@@ -270,6 +270,13 @@ void ieee802154_init(struct net_if *iface);
 #define ieee802154_init(_iface_)
 #endif /* CONFIG_IEEE802154_RAW_MODE */
 
+/**
+ * @brief Store EUI64 value under given location
+ *
+ * @param eui64 An address where EUI64 value should be stored
+ */
+void ieee802154_radio_get_eui64(uint8_t *eui64);
```

Comment:
what this solution differs from ieee802154 net_mgmt GET ext addr?

Plus, that's a lone function that does not really fit into current architecture where generic set/get stuff are going through proper API. see ieee802154 radio api of net_mgmt one

Reply:
Noticed as well that doing this is not going to work if you have 2 different ieee802154 drivers.

Reply:
Yes it won't work for 2 different drivers. I didn't assume it as mandatory use case, so if solution should support more than one driver at once - it's probably not the proper one.